content/posts/blog/first-paris-xmpp-meetup.md

```diff
@@ -6,8 +6,8 @@ blog_id: blog
 category: Summits, Conferences, And Meetups
 ---
 
-After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
+After Berlin, Stockholm, and London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
 
-The Paris meetup is very simply organised by the JabberFr community, as this will be mostly oriented discussions our the modernity and contemporaneity of XMPP/Jabber.
+The Paris meetup is simply organised by the JabberFr community, as this will be mostly oriented discussions on the modernity of XMPP/Jabber.
```

Comment:
> Without waste, very few preparation, nothing fancy like a slidedeck, just two subjects and the open discussions that go around.

Yah, that didn't parse for me. I'm not sure that I have a better suggestion; I'd just leave out "simply" personally.

> on the modern

"on modern XMPP/Jabber" (no article necessary)